src/entity/user/User.ts

```diff
@@ -412,10 +412,10 @@ export const validateUserUpdate = async (
 
 export enum AcquisitionChannel {
   friend = 'friend',
-  social_media = 'social_media',
+  instagram_facebook = 'instagram_facebook',
+  x = 'x',
+  tiktok = 'tiktok',
   search_engine = 'search_engine',
-  blog = 'blog',
-  extension_store = 'extension_store',
```

Comment:
I think it would be better to keep the old ones on the API card for back compat and also since we have those values in DB already. On frontend we can remove them yes 👍

Reply:
I agree. I'd even go more extreme and remove the enum all together from the API and just keep it on the frontend. But it's not a blocker, just easy to do changes in one place only

Reply:
Enum is nice for equality checks. We could fix these kind of double maintenance with commons lib which could be used by apps and api, kinda like `schema` repo but for types and possibly some utils.

Reply:
Agree on Ante's idea here, leaving this here ATM

Reply:
API doesn't really do anything with that, it just receives and stores pretty dumb. There's no use in even maintaining that on the API side even as a shared library. It impacts productivity for now real benefit. That would be my two cents. As mentioned, not a blocker just something to think about